features-json/same-site-cookie-attribute.json

```diff
@@ -19,7 +19,15 @@
     {
       "url":"https://wpdev.uservoice.com/forums/257854-microsoft-edge-developer/suggestions/17140412-support-samesite-cookie-option",
       "title":"Microsoft Edge feature request on UserVoice"
-    }
+    },
+    {
+      "url":"https://developer.microsoft.com/en-us/microsoft-edge/platform/status/samesitecookies/",
+      "title":"Microsoft Edge Browser Status"
+    },
+    {
+      "url":"https://blogs.windows.com/msedgedev/2018/05/17/samesite-cookies-microsoft-edge-internet-explorer/",
+      "title":"MS Edge dev blog: \"Previewing support for same-site cookies in Microsoft Edge\""
+    },
```

Comment:
No comma is required here. Remove it to be correct

Reply:
Fixed